demos/CMakeLists.txt

```diff
@@ -35,11 +35,45 @@ function(check_aws_credentials demo_name)
     endif()
 endfunction()
 
+function(copy_certs_into_target_dir demo_target)
+
+endfunction()
+
+if(DOWNLOAD_CERTS)
+    # Download the Amazon Root CA certificate.
+    message( "Downloading the Amazon Root CA certificate..." )
+    file(MAKE_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/certificates)
+    execute_process(
+        COMMAND curl --url https://www.amazontrust.com/repository/AmazonRootCA1.pem
+        -o ${CMAKE_CURRENT_LIST_DIR}/certificates/AmazonRootCA1.crt
+    )
+    # Download the Mosquitto Root CA certificate.
+    message( "Downloading the Mosquitto Root CA certificate..." )
+    execute_process(
+        COMMAND curl --url https://test.mosquitto.org/ssl/mosquitto.org.crt
+        -o ${CMAKE_CURRENT_LIST_DIR}/certificates/mosquitto.org.crt
+    )
```

Comment:
If our plan is to not use test.mosquitto.org by default any more, we might as well just remove this now.